src/Heading.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
```

Comment:
Good job extracting the heading!

Some small suggestions;

- The most common convention for naming css classes is to use all lower case and have - between words. I would change the class name you use here to `app-header` to follow the convention.
- Don't forget to add an `alt` attribute onto your image. If you open devtools in the browser you will see that there are a few error being thrown from your code. One of them is: 
```
img elements must have an alt prop, either with meaningful text, or an empty string for decorative images 
```

Reply:
Thanks Louise, I will change to add Alt elements to my images for have a betetr syntax on my code.

Reply:
Your code is still throwing this error when it's run
```
Line 7:7:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
```